src/platform/silabs/rs911x/ble_config.h

```diff
@@ -1 +0,0 @@
-rsi_ble_config.h
```

Comment:
this is changed in my PR 

Now there is only no ble_config , ble_config.h is a symbolic link to rs_ble_config.h.

Add your changes there only inside #ifdef